src/test/bin/quic_gtest.cpp

```diff
@@ -1967,6 +1967,18 @@ TEST_P(WithAbortiveArgs, AbortiveShutdown) {
     }
 }
 
+TEST_P(WithCancelOnLossArgs, CancelOnLossSend) {
```

Comment:
This needs to be protected by `QUIC_TEST_DATAPATH_HOOKS_ENABLED` just like the `QuicTestKeyUpdateRandomLoss` test above. Please follow that pattern for all the test references.

P.S. This is because those test hooks are not enabled for Release builds (security hole), and as a result we have to do a bit of shuffling of test code in Release mode.